ext/SolidStateDetectorsGeant4Ext.jl

```diff
@@ -4,32 +4,92 @@ module SolidStateDetectorsGeant4Ext
 
 @static if isdefined(Base, :get_extension)
     using Geant4
+    using LightXML
 else
     using ..Geant4
+    using ..LightXML
 end
 
-import SolidStateDetectors
 
+include("io_gdml.jl")
+using Suppressor
 
-function _ssdsim2gdml(io::IO, sim::SolidStateDetectors.Simulation)
-    # Dummy implementation:
-
-    println(io, """<?xml version="1.0" encoding="UTF-8" standalone="no" ?>""")
-    println(io, """<gdml>""")
-    println(io, """</gdml>""")
-end
-
+# Given an SSD simulation object, create corresponding GDML file to desired location
+function Geant4.G4JLDetector(sim::SolidStateDetectors.Simulation, output_filename::String = "tmp.gdml"; verbose::Bool = true)
```

Comment:
`G4JLDetector` is an abstract type in `Geant4.jl`.
Should we rename this to `G4JLDetectorSSD` to stick to the notation `G4JLDetectorGDML` when reading `GDML` files?